src/linear_algebra/basic.lean

```diff
@@ -1176,6 +1201,31 @@ le_antisymm
   (supr_le $ assume i, subset.trans (assume m hm, set.mem_Union.mpr ⟨i, hm⟩) subset_span)
   (span_le.mpr $ Union_subset_iff.mpr $ assume i m hm, mem_supr_of_mem i hm)
 
+lemma supr_to_add_submonoid {ι : Sort*} (p : ι → submodule R M) :
+  (⨆ i, p i).to_add_submonoid = ⨆ i, (p i).to_add_submonoid :=
+begin
+  apply le_antisymm,
```

Comment:
Is it ridiculous to golf like this and then drop the `{`, `}` below?
```suggestion
  refine le_antisymm _ (supr_le (λ i, to_add_submonoid_mono (le_supr _ i))),
```

Reply:
Looking at the rest of the file, this seems to fit the style - adopted with `$`s instead of `)))`.